parity/cli/mod.rs

```diff
@@ -238,6 +238,11 @@ usage! {
 	{
 		// Global flags and arguments
 		["Operating Options"]
+			FLAG flag_allow_empty_block_result: (bool) = false, or |c: &Config| c.parity.as_ref()?.allow_empty_block_result
```

Comment:
please move it to RPC section and call it `--jsonrpc-allow-missing-blocks`, maybe?